workflow-service/src/test/java/com/redhat/parodos/project/controller/ProjectControllerTest.java

```diff
@@ -187,6 +188,22 @@ public void testRemoveUsersFromProject() throws Exception {
 		verify(projectService, times(1)).removeUsersFromProject(any(), any());
 	}
 
+	@Test
+	public void testRemoveUsersFromProject_WithEmptyUsersList() throws Exception {
+		// Given
+		List<String> users = List.of();
+		ProjectUserRoleResponseDTO projectUserRoleResponseDTO = ProjectUserRoleResponseDTO.builder()
+				.projectName("test-project").build();
+
+		// When
+		mockMvc.perform(
+						this.deleteRequestWithValidCredentials(String.format("/api/v1/projects/%s/users", UUID.randomUUID()))
+								.content(JSONObject.valueToString(users)))
+				.andExpect(MockMvcResultMatchers.status().isBadRequest());
```

Comment:
nit: this looks bad. Do you think it's possible to improve it? 
I know that it's formatter fault, so feel free to ignore this comment.